diff -uraN xfce4-panel-profiles-1.0.14/xfce4-panel-profiles/xfce4-panel-profiles.py omv-xfce4-panel-profiles-1.0.14/xfce4-panel-profiles/xfce4-panel-profiles.py
--- xfce4-panel-profiles-1.0.14/xfce4-panel-profiles/xfce4-panel-profiles.py	2023-06-05 09:01:02.000000000 +0200
+++ omv-xfce4-panel-profiles-1.0.14/xfce4-panel-profiles/xfce4-panel-profiles.py	2023-06-26 13:37:24.947228695 +0200
@@ -28,18 +28,11 @@
 
 import gi
 gi.require_version('Gtk', '3.0')
-# Try to import the new Libxfce4ui gir name (since 4.15.7)
-# if it does not exists, try the old libxfce4ui
-try:
-  gi.require_version('Libxfce4ui', '2.0')
-  from gi.repository import Libxfce4ui as libxfce4ui
-  from gi.repository import Libxfce4util as libxfce4util
-except ValueError:
-  gi.require_version('libxfce4ui', '2.0')
-  from gi.repository import libxfce4ui
-  from gi.repository import libxfce4util
+gi.require_version('Libxfce4ui', '2.0')
 
-from gi.repository import Gtk, GLib, Gio
+from gi.repository import libxfce4util
+
+from gi.repository import Gtk, GLib, Gio, Libxfce4ui
 
 from panelconfig import PanelConfig
 
